mpeg2/src/temi.rs

```diff
@@ -0,0 +1,275 @@
+use crate::{DecodeError, EncodeError};
+use byteorder::{BigEndian, ByteOrder};
+use core2::io::Write;
+
+const AF_DESCR_TAG_TIMELINE: u8 = 0x04;
+
+#[derive(Debug, Default, PartialEq)]
+pub struct TEMITimelineDescriptor {
+    pub has_timestamp: Option<u8>,
+    pub timescale: u32,
+    pub media_timestamp: u64,
+
+    pub ntp_timestamp: Option<u64>,
+    pub ptp_timestamp: Option<u128>,
+
+    pub has_timecode: Option<u8>,
+    pub drop: bool,
+    pub frames_per_tc_seconds: u16,
+    pub duration: u16,
+    pub time_code: u64,
+
+    pub force_reload: Option<bool>,
+    pub paused: Option<bool>,
+    pub discontinuity: Option<bool>,
```

Comment:
Why not just make these `bool`s?

```suggestion
    pub force_reload: bool,
    pub paused: bool,
    pub discontinuity: bool,
```

Reply:
I don't see any reason to use `bool` instead of `Option<bool>`.  Encoded `None` valued won't be decoded to `None`.    Will remove `Option`.